api/coreservice.go

```diff
@@ -613,6 +616,28 @@ func (core *coreService) SuggestGasPrice() (uint64, error) {
 	return core.gs.SuggestGasPrice()
 }
 
+func (core *coreService) SuggestTipCap() (*big.Int, error) {
+	sp, err := core.SuggestGasPrice()
```

Comment:
does Eth have a `SuggestedGasFeeCap()` and what is the implementation?
since `effective gas = min(tip, feeCap - baseFee)`, should use `feeCap - baseFee` as the suggested value?

Reply:
the impl of eth is https://github.com/ethereum/go-ethereum/blob/master/eth/gasprice/gasprice.go#L153.

In general, it calculates the final priorityfee of some recent transactions and takes a certain value in them. We can currently simplify the implementation, and further optimizations can be made later.